sql-plugin-api/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

```diff
@@ -44,7 +44,7 @@ import org.apache.spark.util.MutableURLClassLoader
     3. a smaller fraction of classes that differ under one of the supported Spark versions
     com/nvidia/spark/SQLPlugin.class
     spark-shared/com/nvidia/spark/rapids/CastExprMeta.class
-    spark311/org/apache/spark/sql/rapids/GpuUnaryMinus.class
+    spark320/org/apache/spark/sql/rapids/GpuUnaryMinus.class
```

Comment:
This change doesn't make sense given the context.  This line is now duplicated with the one below.  Probably need to pick a different example class to use here (can examine the contents of a dist jar after a buildall for a class that has a spark320 version and one other spark version)

Reply:
Sounds good to me, GpuLast.class should be good to replace current class here

`Binary files ./spark331/org/apache/spark/sql/rapids/aggregate/GpuLast.class and ./spark320/org/apache/spark/sql/rapids/aggregate/GpuLast.class differ`